### ISSUE TITLE
Make libX11 dependency optional

### ISSUE THREAD

**Issue by SkeLLLa** [justadudewhohacks/face-recognition.js#70]
Hi. I saw that in some issues you've mentioned that GUI dependency could be optional. That will be great and it will also allow reducing docker images size.

**Comment by justadudewhohacks** [comment]
Yes that's possible and planned to be added a quite some time ago now. Unfortunately didn't find the time to provide this option yet.
